pcm_NR.m

```diff
@@ -35,8 +35,11 @@
 
 % Variable argument otions
 %--------------------------------------------------------------------------
+try
 OPT=rsa.getUserOptions(varargin,OPT,{'HessReg','thres','low','numIter','verbose','regularization'});
-
+catch
+    error('This might need the RSA toolbox to work. Did you had it to the matlab path?')

```

Comment:
wording
"do you have it in the matlab path"
"did you install it in the matlab path"

Reply:
Oh good lord! I seem to be unable to write a proper English sentence! Will fix that... Let's hope I don't put some warning in French instead.